src/components/details/add-to-cart.tsx

```diff
@@ -18,9 +19,10 @@ export const AddToCartSection = ({
   quantity,
   setQuantity,
   buy,
+  stock,
 }: AddToCartSectionProps) => {
   const numberItems: DropdownItem[] = Array.from(
```

Comment:
Use intuitive and descriptive variable names to enhance code readability and maintainability for future developers. For example, consider renaming this variable to something like `availableQuantityOptions` to make its purpose clearer at a glance.